app/services/katello/pulp3/repository/ansible_collection.rb

```diff
@@ -1,72 +1,43 @@
-require 'pulp_file_client'
 require 'pulp_ansible_client'
 
 module Katello
   module Pulp3
     class Repository
       class AnsibleCollection < ::Katello::Pulp3::Repository
-        def create_content_remote
-          remote_collection_data = PulpAnsibleClient::CollectionRemote.new(remote_options)
-          response = pulp3_api.remotes_ansible_collection_create(remote_collection_data)
-          response._href
+        def api_client
+          PulpAnsibleClient::ApiClient.new(smart_proxy.pulp3_configuration(PulpAnsibleClient::Configuration))
```

Comment:
This should probably mimic the file service method: https://github.com/Katello/katello/pull/8190/files#diff-300f565091a88861433c44ce4f5005e7R11

Reply:
strange that this didn't trigger any test failures

Reply:
fixed!

Reply:
Yes..I saw this on indexing..CRUD worked fine..We don't have collection tests for sync/indexing as of now..